bower.json

```diff
@@ -23,6 +23,7 @@
     "datatables": "~1.10.12",
     "chosen": "~1.5.1",
     "fontawesome": "~4.6.3",
-    "leaflet": "^0.7.7"
+    "leaflet": "^0.7.7",
+    "moment": "~2.17.1"
```

Comment:
@Huyeng moment is a javscript library to work with dates, I strongly recomend it to you: http://momentjs.com/

it is used here https://github.com/OpenDevelopmentMekong/wp-odm_theme/blob/3e6eb94b849112ef3df64674b9d037b46cbd7f86/inc/advanced-navigation.php#L213